### PR TITLE
[Yaml] sync test and actually triggered message

### PR DESCRIPTION
**Pull request by xabbuh**
| Q | A |
| --- | --- |
| Branch? | 3.1 |
| Bug fix? | yes |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | see failures in #19757 |
| License | MIT |
| Doc PR |  |

### CODE REVIEW

**Comment by HeahDude** [comment]
👍

Status: reviewed

**Comment by javiereguiluz** [comment]
👍

File: src/Symfony/Component/Yaml/Tests/InlineTest.php [hunk 1]
```diff
@@ -260,7 +260,7 @@ public function getScalarIndicators()
      */
     public function testParseUnquotedScalarStartingWithPercentCharacter()
     {
-        ErrorAssert::assertDeprecationsAreTriggered('Not quoting a scalar starting with the "%" indicator character is deprecated since Symfony 3.1 and will throw a ParseException in 4.0.', function () {
+        ErrorAssert::assertDeprecationsAreTriggered('Not quoting the scalar "%foo " starting with the "%" indicator character is deprecated since Symfony 3.1 and will throw a ParseException in 4.0.', function () {
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Yaml/Tests/InlineTest.php]: @xabbuh is the trailing space legitimate or is it another bug?

**Comment by nicolas-grekas** [comment]
Thank you @xabbuh.

File: src/Symfony/Component/Yaml/Tests/InlineTest.php (hunk 1 above)
Comment (xabbuh) [inline src/Symfony/Component/Yaml/Tests/InlineTest.php]: imo it's legitimate, it's what the parser receives at that stage

**Pull request merged**